exp-player/addon/components/exp-exit-survey.js

```diff
@@ -123,37 +138,47 @@ export default ExpFrameBaseComponent.extend({
             };
             return {
                 schema: this.get('form.schema'),
-                options: newOptions        
+                options: newOptions
             };
         },
         set(_, value) {
             this.set('formSchema', value);
             return value;
         }
-        
     }),
     section1: true,
     formData: [],
     formActions: Ember.computed(function() {
         var root = this;
         return {
-            update: function () {
+            update: function() {
                 this.refreshValidationState(true);
                 if (this.isValid(true)) {
-                   root.set('formData', this.getValue());
-                   root.set('section1', false);
+                    root.set('formData', this.getValue());
+                    root.set('section1', false);
                 }
             },
             finish: function() {
                 var formData = root.get('formData');
                 Ember.merge(formData, this.getValue());
                 root.set('formData', formData);
-                console.log('Post-study survey complete.')
                 root.actions.next();
             }
-        }
+        };
+    }),
+    currentSessionsCompleted: Ember.computed('frameContext', function() {
```

Comment:
This is a computed value of the frameContext (available to every frame).  

If we find a more general use case we can move this code to a mixin. For now it's fine here.